README.md

```diff
@@ -11,6 +11,8 @@
 <!-- TODO: add prev pr status check badge -->
 <!-- TODO: add prod pr status check badge -->
 
+[![Codecov](https://img.shields.io/codecov/c/github/KinsonDigital/Velaptor?color=2F8840&label=Code%20Coverage&logo=codecov)](https://app.codecov.io/gh/KinsonDigital/Partithyst/tree/release%2Fv1.0.0)
```

Comment:
Fix the URL of the INNER markdown.  It is still set to **Velaptor**.  Change this to **Partithyst**.

```suggestion
[![Codecov](https://img.shields.io/codecov/c/github/KinsonDigital/Partithyst?color=2F8840&label=Code%20Coverage&logo=codecov)](https://app.codecov.io/gh/KinsonDigital/Partithyst/tree/release%2Fv1.0.0)
```